src/GoogleAuthenticator.php

```diff
@@ -123,15 +113,6 @@ public function getCode($secret, /* \DateTimeInterface */$time = null): string
         return str_pad((string) ($truncatedHash % $this->pinModulo), $this->passCodeLength, '0', STR_PAD_LEFT);
     }
 
-    /**
-     * NEXT_MAJOR: Remove this method.
-     *
-     * @param string $user
-     * @param string $hostname
-     * @param string $secret
-     *
-     * @deprecated deprecated as of 2.1 and will be removed in 3.0. Use Sonata\GoogleAuthenticator\GoogleQrUrl::generate() instead.
-     */
```

Comment:
You should keep this when removing the method from the interface

Reply:
Fixed, thanks!

Reply:
So, what should we do?